spec.html

```diff
@@ -39912,8 +39912,8 @@ <h1>`Promise.all` Resolve Element Functions</h1>
           <emu-alg>
             1. Let _F_ be the active function object.
             1. Let _alreadyCalled_ be _F_.[[AlreadyCalled]].
-            1. If _alreadyCalled_.[[Value]] is *true*, return *undefined*.
-            1. Set _alreadyCalled_.[[Value]] to *true*.
+            1. If _alreadyCalled_ is *true*, return *undefined*.
+            1. Set _alreadyCalled_ to *true*.
```

Comment:
Since `alreadyCalled` is now a primitive, setting it won't affect `F.[[AlreadyCalled]]`, right?
I believe `F.[[AlreadyCalled]]` should be assigned to directly.